packages/react-compose/src/types.ts

```diff
@@ -51,6 +70,9 @@ export type ComposeOptions<InputProps = {}, InputStylesProps = {}, ParentStylesP
 
   handledProps?: (keyof InputProps | 'as')[];
   overrideStyles?: boolean;
+
+  slots?: Record<string, React.ElementType>;
+  slotProps?: Record<string, (props: Extendable<InputProps>) => object>;
```

Comment:
This is hidden any, let's avoid it until it's really necessary

Reply:
`slotProps?: Record<string, (props: InputProps) => object>;`
Like this you mean?

Reply:
Or you are talking about `Record`?

Reply:
Yep